tools/travis/run_lxd_container.sh

```diff
@@ -32,7 +32,7 @@ name="$1"
 lxc="/snap/bin/lxc"
 
 echo "Starting the LXD container."
-$lxc launch --ephemeral ubuntu:xenial "$name"
+$lxc launch --ephemeral --config security.privileged=true ubuntu:xenial "$name"
```

Comment:
Why do we need privileged containers here?

Reply:
The commit that added this has the info, but here it is: https://bugs.launchpad.net/snapd/+bug/1709536

Reply:
Ah yes, I hit that myself. I fixed it by removing the Nice value from the systemd unit file though as opposed to making the container privileged. It's probably not a big deal, let's just keep an eye on the bug and change back to unprivileged once it's fixed.